master/getting-started/docker/index.md

```diff
@@ -1,4 +1,47 @@
 ---
 title: Calico with Docker
 ---
-More information coming soon!
+
+Calico implements a Docker network plugin that can be used to provide routing and advanced network policy for Docker containers.
+
+Use the navigation bar on the left to view information on Calico for Docker,
+or continue reading for an overview of recommended guides to get started.
+
+
+## Installation
+
+#### [Requirements](installation/requirements)
+
+Information on running etcd and configuring Docker for multi-host networking.
```

Comment:
I'm not so sure. My long term plan for the Requirements guide is to improve instructions for starting etcd and configuring a docker cluster store, since those are huge pain points for new users. I don't want to bloat up the Install guide with that kind of information.